ForceGamaAttack/src/entities/BulletManager.java

```diff
@@ -0,0 +1,48 @@
+package entities;
+
+import java.util.ArrayList;
+import java.util.LinkedList;
+
+import jplay.Window;
+
+public class BulletManager {
+	LinkedList<Bullet> objects = new LinkedList<Bullet>();
+	
+	public void addBullet(double x, double y, int floor) {
+		Bullet bullet = new Bullet(x, y);
+		bullet.setFloor(floor);
+		
+		objects.addFirst(bullet);
+		System.out.println("ADD");
+	}
+	
+	public void step(int floor){
+		for(int i = 0; i < objects.size(); i++) {
+			Bullet bullet = objects.removeFirst();
+			System.out.println("Step");
```

Comment:
Eu sei que tá todo mundo esquecendo disso kkkkk mas vamo tirar os print de debug quando for abrir PR

Reply:
Prints de debug removidos.